scripts/build_utils.sh

```diff
@@ -1850,3 +1850,42 @@ function set_centos_python3_version() {
     sudo dnf reinstall -y $EXPECTED_PYTHON3_VERSION || sudo dnf install -y $EXPECTED_PYTHON3_VERSION
     sudo ln -fs /usr/bin/$EXPECTED_PYTHON3_VERSION /usr/bin/python3
 }
+
+function report_github_pr_check_status() {
```

Comment:
can this go in a separate file?

Reply:
In `scripts/`?   I suppose.  I'm not sure what the point of a massive `build_utils.sh` file was anyway.

Reply:
yeah, I'd suggest adding a new file in `scripts/`, maybe even named after this function. A snippet like this can make it both executable and source-able: https://github.com/ceph/ceph-build/blob/pipeline-container/scripts/update_shaman.sh#L48-L51

Reply:
The point was simplicity; "one library"  Bash is not a good modular language.

Reply:
Done